.github/workflows/check-dist.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           fi
 
       # If dist/ was different than expected, upload the expected version as an artifact
-      - uses: actions/upload-artifact@v2
+      - uses: actions/upload-artifact@v4
```

Comment:
🤔 Why do we preserve this? Maybe I should delete this. The diff is printed above and we will have the logs from the run.